lua/ulx/modules/sh/yeet.lua

```diff
@@ -0,0 +1,38 @@
+local CATEGORY_NAME = "Fun"
+
+local MIN_STRENGTH = 6000	
+local MAX_STRENGTH = 500000	
+local DEFAULT_STRENGTH = 6000 
```

Comment:
ULX allows you to set limits on the parameters provided. Setting them here makes it kind of restrictive. If some other server wanted to use a higher max or a lower min, they wouldn't be able to set that in the ULX menu.

I'm not sure what the "right" solution is here, especially because I don't know the significance of these values, so just keep that in mind.